bbc/simorgh
# Psammead package bumps

Resolves https://github.com/bbc/simorgh/issues/2071

**Overall change:** Bump all psammead components

**Code changes:**

- @bbc/psammead-assets: https://github.com/bbc/psammead/pull/704 Remove fill from SVGs
- @bbc/psammead-image: https://github.com/bbc/psammead/pull/690 Updates implementation of image fade in to additionally transition visibility
- @bbc/psammead-story-promo: https://github.com/bbc/psammead/pull/694 Fix floating media indicator and promo link bugs

The following components bumped psammead but based on the snapshot tests there should be no affect
```
@bbc/psammead-caption
@bbc/psammead-copyright
@bbc/psammead-figure
@bbc/psammead-media-indicator
@bbc/psammead-paragraph
@bbc/psammead-section-label
@bbc/psammead-sitewide-links
@bbc/psammead-story-promo
@bbc/psammead-story-promo-list
@bbc/psammead-timestamp-container
```

**Testing notes:**
- Ensure https://github.com/bbc/simorgh/issues/2071 is fixed
- Regression lazy load image animation on articles
- Ensure https://github.com/bbc/psammead/issues/699 is fixed

---
